### ISSUE TITLE
main and develop branch

### ISSUE THREAD

**Issue by jwboth** [pmgbergen/DarIA#60]
We should consider introducing another stable branch called develop. When starting to have users at IFT, they will be very unhappy when we are introducing major changes which break their scripts. Therefore introduing another layer and actualy also tags should be considered. Maybe not immediately, but in the near future.

**Comment by EStorvik** [comment]
I agree! I am not sure if anyone (but perhaps Benyamine) has started using it yet though... I thought he would send me an email, but I never got one. 
Nevertheless, we should definitely have a develop branch!

**Comment by jwboth** [comment]
A new suggestion: We keep everything as it is right now and introduce a develop branch after release.
